setup.py

```diff
@@ -45,8 +45,9 @@
     install_requires=[
         "icontract >=2.3.4, <3",
         "pydotplus >=2, <3",
-        "pyparsing >=2.4.0, <3",
         "z3-solver >=4, <5",
+        "RecordFlux-language@git+https://github.com/Componolit/RecordFlux-language.git"
+        "@master#egg=RecordFlux-language",
```

Comment:
Shouldn't this point to a specific commit?

Reply:
Yes, pinned to a version tag in 92e253348e29d51707aa1da92bd7734625af61e8.